src/components/manage/Controlpanels/UsersControlpanel.jsx

```diff
@@ -804,6 +804,7 @@ class UsersControlpanel extends Component {
                     messages.searchUsers,
                   )}
                   onChange={this.onChangeSearch}
+                  id="user-search-input"
```

Comment:
There is this ongoing discussion of ids vs classes in CSS. Maybe @sneridagh wants to weight in or we want to have a brief discussion about it.